ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/tabsontop/v1/tabsontop/_cq_dialog/.content.xml

```diff
@@ -34,6 +34,56 @@
+                                <items jcr:primaryType="nt:unstructured">
+                                    <column
+                                            jcr:primaryType="nt:unstructured"
+                                            sling:resourceType="granite/ui/components/coral/foundation/container">
+                                        <items jcr:primaryType="nt:unstructured">
+                                            <repeatableText
+                                                    jcr:primaryType="nt:unstructured"
+                                                    sling:resourceType="granite/ui/components/coral/foundation/text"
+                                                    text="Use the make tabs on top repeatable option to allow form users to dynamically add multiple panels to the form."/>
+                                            <repeatable
+                                                    jcr:primaryType="nt:unstructured"
+                                                    wrapperClass="cmp-adaptiveform-panelcontainer__repeatable"
+                                                    sling:resourceType="granite/ui/components/coral/foundation/form/switch"
+                                                    name="./repeatable"
+                                                    fieldLabel="Make tabs on top repeatable"
+                                                    checked="false"/>
+                                            <minOccur
+                                                    jcr:primaryType="nt:unstructured"
+                                                    wrapperClass="cmp-adaptiveform-panelcontainer__minOccur"
+                                                    sling:resourceType="granite/ui/components/coral/foundation/form/numberfield"
+                                                    defaultValue="1"
+                                                    fieldLabel="Minimum repetitions"
+                                                    fieldDescription="Minimum number of times the tabs on top appears on the form"
+                                                    min="0"
+                                                    name="./minOccur"/>
+                                            <maxOccur
+                                                    jcr:primaryType="nt:unstructured"
+                                                    wrapperClass="cmp-adaptiveform-panelcontainer__maxOccur"
+                                                    sling:resourceType="granite/ui/components/coral/foundation/form/numberfield"
+                                                    emptyText="No Limit"
+                                                    fieldLabel="Maximum repetitions"
+                                                    fieldDescription="Maximum number of times the tabs on top can appear on the form"
+                                                    min="1"
```

Comment:
maxOccur can be -1, right?

Reply:
We have added placeholder as 'No Limit', so if nothing is specified or is '-1' then place holder text shows up.